build.rs

```diff
@@ -0,0 +1,7 @@
+//! Emits the necessary "checked" compile time flag warning for "loom".
+//!
+//! Without this, the compiler complains about the unspecified "loom" flag.
+
+fn main() {
+    println!("cargo:rustc-check-cfg=cfg(loom)");
```

Comment:
I would like to use https://github.com/rust-lang/cargo/pull/13913 's config option instead of adding build script.

Reply:
Tracked in #136